lib/web/apps.go

```diff
@@ -362,3 +356,20 @@ func (h *Handler) proxyDNSNames() (dnsNames []string) {
 	}
 	return dnsNames
 }
+
+// extractAppsWithoutTCPEndpoint iterates over a list of app servers and extracts
+// its application without its URI prefixed with "tcp". It also keeps a
+// counter of how many TCP applications were excluded from the list (e.g. used to
+// subtract it from the TotalCount received from ListResources api).
+func extractAppsWithoutTCPEndpoint(appServers []types.AppServer) (apps types.Apps, numExcluded int) {
```

Comment:
```suggestion
// removeTCPApps filters TCP apps our of the list of app servers.
// TCP apps are filtered out because they are not accessible from the web UI.
// It returns the HTTP apps and the number of TCP apps that were removed.
func removeTCPApps(appServers []types.AppServer) (apps types.Apps, numExcluded int) {
```

Reply:
I edited Zac's suggestion to fix a typo.